### PR TITLE
repository: make it work

### PR DESCRIPTION
**Pull request by Migorithm**
### Issue Link

-

### Description

-

### Changes

- make repository work by changing pooling function

### CODE REVIEW

File: src/adapters/repositories/account.rs [hunk 1]
```diff
@@ -29,12 +29,12 @@ impl AccountRepository for SqlRepository {
             WHERE name = $1
         "#,
         email)
-            .fetch_one(pool())
+            .fetch_one(pool().await)
```
Comment (seonwoo960000) [inline src/adapters/repositories/account.rs:32]: `pool().await` or `self.pool`? How about we unify how we reference the pool?

File: migrations/20231111124304_schema.up.sql [hunk 2]
```diff
@@ -11,8 +11,8 @@ CREATE TABLE IF NOT EXISTS public.account
     name            text                                     not null,
     status          account_status DEFAULT 'active'          not null,
     hashed_password text                                     not null,
-    created_at      timestamp      default CURRENT_TIMESTAMP not null,
-    updated_at      timestamp      default CURRENT_TIMESTAMP not null
+    created_at      timestamptz      default CURRENT_TIMESTAMP not null,
```
Comment (seonwoo960000) [inline migrations/20231111124304_schema.up.sql:14]: Why not UTC? We may keep time as UTC in the backend and just translate those at frontend

File: src/adapters/repositories/account.rs (hunk 1 above)
Comment (Migorithm) [reply src/adapters/repositories/account.rs]: @seonwoo960000 done

File: migrations/20231111124304_schema.up.sql (hunk 2 above)
Comment (Migorithm) [reply migrations/20231111124304_schema.up.sql:14]: @seonwoo960000 that's exactly what I want to do. 

initial implementation was naive time which was not time-aware.  I changed it to time-aware field which is UTC.

Comment (seonwoo960000) [reply migrations/20231111124304_schema.up.sql:14]: Why do we need to use `timestamptz` to store extra timezone information if our purpose is to store UTC only?

**Pull request merged**